test/minter-suite-minters/DA/MinterDAExp/MinterDAExpV3.test.ts

```diff
@@ -0,0 +1,306 @@
+import {
+  BN,
+  constants,
+  expectEvent,
+  expectRevert,
+  balance,
+  ether,
+} from "@openzeppelin/test-helpers";
+
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { ethers } from "hardhat";
+
+import {
+  getAccounts,
+  assignDefaultConstants,
+  deployAndGet,
+  deployCoreWithMinterFilter,
+  safeAddProject,
+} from "../../../util/common";
+import { ONE_MINUTE, ONE_HOUR, ONE_DAY } from "../../../util/constants";
+import { MinterDAExp_Common } from "./MinterDAExp.common";
+import { MinterDAV1V2_Common } from "../MinterDAV1V2.common";
+
```

Comment:
Please add the following to all minter tests to prevent nuisance logs about event overloading:
```js
// hide nuisance logs about event overloading
Logger.setLogLevel(Logger.levels.ERROR);
```